addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -207,7 +207,7 @@
                                     <field name="stage_id" domain="[('user_id', '=', False), ('project_ids', 'in', [project_id])]"/>
                                     <button name="action_open_task" type="object" title="View Task" string="View Task" class="btn btn-link float-end"
                                             context="{'form_view_ref': 'project.project_sharing_project_task_view_form', 'search_view_ref': 'project.project_sharing_project_task_view_search'}"
-                                            invisible="project_id or project_id != id"/>
+                                            invisible="not id"/>
```

Comment:
The fix is not right, we should keep the initial condition but we should use the active_id instead of id.

```suggestion
                                            invisible="project_id != context.get('active_id')"/>
```

I guess the active_id is now in the context.